HISTORY.rst

```diff
@@ -2,6 +2,57 @@
 History
 =======
 
+3.24.0 (2020-11-30)
+-------------------
+
+Critical Changes:
+
+- The flows ``dev_org``, ``dev_org_namespaced``, ``qa_org``, ``ci_feature``, and ``install_beta`` now run the ``update_dependencies`` task with the ``include_beta`` option enabled, so dependencies will be installed using the most recent beta release instead of the most recent final release. The ``beta_dependencies`` flow is no longer used and is considered deprecated.
+
+- The flows ``ci_feature_beta_deps`` and ``dev_org_beta_deps`` are now deprecated and should be replaced by their default equivalents above.
+
+- The ``ci_feature_2gp`` flow has been changed to use ``config_apextest`` instead of ``config_managed`` to avoid configuration steps that are unnecessary for running Apex tests. This means that in order for ``ci_feature_2gp`` to work, ``config_apextest`` must be set up to work in both managed and unmanaged contexts.
+
+- When connecting GitHub using ``cci service connect github``, we now prompt for a personal access token instead of a password. (GitHub has removed support for accessing the API using a password as of November 2020.) If you already had a token stored in the ``password`` field, it will be transparently migrated to ``token``. If you were specifying `--password` on the command line when running this command, you need to switch to `--token` instead.
```

Comment:
```suggestion
- When connecting GitHub using ``cci service connect github``, we now prompt for a personal access token instead of a password. (GitHub has removed support for accessing the API using a password as of November 2020.) If you already had a token stored in the ``password`` field, it will be transparently migrated to ``token``. If you were specifying ``--password`` on the command line when running this command, you need to switch to ``--token`` instead.
```